roles/create-jira-project/tasks/create_project.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Create Sample Project
+  uri:
+    url: "{{ atlassian_url }}/rest/api/2/project"
+    method: POST
+    user: "{{ atlassian_username }}"
+    password: "{{ atlassian_password }}"
+    return_content: yes
+    force_basic_auth: yes
+    body_format: json
+    header:
+      - Accept: 'application/json'
+      - Content-Type: 'application/json'
+    body: "{ 'key': '{{ atlassion_key }}',
+             'name': '{{ atlassion_project }}',
+             'projectTypeKey': 'software',
+             'projectTemplateKey': 'com.pyxis.greenhopper.jira:gh-simplified-scrum',
+             'description': 'Project created using Ansible playbook',
+             'lead': '{{ atlassian_username }}',
+             'assigneeType': 'PROJECT_LEAD',
+             'avatarId': 10200,
+             'permissionScheme': '{{ PermissionScheme }}',
+             'notificationScheme': 10000, 
+             'categoryId': '{{ CategoryID }}' 
+             }"
+    status_code: 201         
+  register: result
+
+- name: Project Creation Details
```

Comment:
Please remove debugs, or otherwise set them with a verbosity level to only print of verbose level is 1 or greater.